docs/configuration/tooltip.md

```diff
@@ -83,29 +84,29 @@ Allows filtering of [tooltip items](#tooltip-item-interface). Must implement at
 
 ## Tooltip Callbacks
 
-The tooltip label configuration is nested below the tooltip configuration using the `callbacks` key. The tooltip has the following callbacks for providing text. For all functions, 'this' will be the tooltip object created from the Chart.Tooltip constructor.
+The tooltip label configuration is nested below the tooltip configuration using the `callbacks` key. The tooltip has the following callbacks for providing text. For all functions, `this` will be the tooltip object created from the `Chart.Tooltip` constructor.
 
 All functions are called with the same arguments: a [tooltip item](#tooltip-item-interface) and the data object passed to the chart. All functions must return either a string or an array of strings. Arrays of strings are treated as multiple lines of text.
 
 | Name | Arguments | Description
 | ---- | --------- | -----------
-| `beforeTitle` | `Array[tooltipItem], data` | Returns the text to render before the title.
-| `title` | `Array[tooltipItem], data` | Returns text to render as the title of the tooltip.
-| `afterTitle` | `Array[tooltipItem], data` | Returns text to render after the title.
-| `beforeBody` | `Array[tooltipItem], data` | Returns text to render before the body section.
+| `beforeTitle` | `tooltipItem[], data` | Returns the text to render before the title.
```

Comment:
What do you think about `TooltipItem[]` since it's an interface?

Reply:
I didn’t change after considering between `tooltipItem[]` and `TooltipItem[]`, but if `TooltipItem[]` seems to make more sense, I’m fine with that.

Reply:
We used to capitalize classes and interfaces and I think `TooltipItem` is an interface here.

Reply:
I'm on on-board with capitalizing since it's an interface that we've publicly defined. The underlying items are just plain objects though.